scripts/jenkins/qa_devstack.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+
+# Use ipv4 by default to be backwards compatible
+if [ "$1" == "ipv6" ]; then
+    SVC_IP_VERSION=6
+else
+    SVC_IP_VERSION=4
+fi
```

Comment:
We try to move default values out of if and switch/case statements (they easily get lost when changing the if/switch later), and as bash is used the if can be shortened as well:
```
SVC_IP_VERSION=4
[[ $1 == "ipv6" ]] && SVC_IP_VERSION=6
```